spec/stubs/business.rb

```diff
@@ -0,0 +1,471 @@
+module Stubs
+  class Business
+    class << self
+      def success_create(params)
+        response_body = {
+          id: 1,
+          type: "business",
+          data: {
+            id: "biz_xyz",
+            platform_account_id: "acc_xyz",
+            legal_name: "Business Name",
+            website_url: "https://justifi.ai",
+            email: "business@justifi.ai",
+            phone: "6124011111",
+            doing_business_as: "Best Business",
+            business_type: "for_profit",
+            business_structure: "sole_proprietorship",
+            classification: "government limited non_profit partnership corporation public_company sole_proprietor",
```

Comment:
the `classification` is a combination of `business_type` and `business_structure`, so in this case the correct value should be `sole_proprietor`

```suggestion
            classification: "sole_proprietor",
```